### ISSUE TITLE
Preview scroll breaks on fenced comments

### ISSUE THREAD

**Issue by johnchristopherjones** [mark-hahn/markdown-scroll-sync#25]
Scroll syncing breaks when fenced code blocks are used with inline comments.  For example, pasting this about midway into `test.md` breaks scroll sync.

    ```javascript
    // js
    myModule.controller('MainCtrl', function() {
      this.title = 'Hello, World.';
    });
    ```

This appears to be caused in `startTracking:` by a selector that selects `.line` and later expects all the matching elements to have a data-screen-row attribute.  However, the inline `// js` comment above is nested within a `<span class="line">` element.

This could be fixed by changing the selector.  Changing the selector to `div.line` or `div[data-screen-row]` appears to do the trick.

**Comment by mark-hahn** [comment]
Thank you very much for the trouble if figuring this out.  Can you please
do a pull request?

**Comment by johnchristopherjones** [comment]
I'm still trying to figure out the workflow.  Does that do it?  Can I/should I open a pull request without opening an issue?

**Comment by mark-hahn** [comment]
I don't know what you did but it worked.

**Comment by mark-hahn** [comment]
I've released a new version 2.0.1 that is a total rewrite.  I think it fixes all issues and works much better than before.  Please try it.
